rcl/src/rcl/subscription.c

```diff
@@ -233,15 +234,19 @@ rcl_subscription_get_default_options()
   default_options.rmw_subscription_options = rmw_get_default_subscription_options();
 
   // Load disable flag to LoanedMessage via environmental variable.
-  bool disable_loaned_message = false;
-  rcl_ret_t ret = rcl_get_disable_loaned_message(&disable_loaned_message);
-  if (ret == RCL_RET_OK) {
-    default_options.disable_loaned_message = disable_loaned_message;
-  } else {
+  // TODO(clalancette): This is kind of a copy of rcl_get_disable_loaned_message(), but we need
+  // more information than that function provides.
```

Comment:
yeah, maybe we can make it exclusive either 0 (false) or 1 (true) later. if that neither of them, fallback to endpoint default.